typing/persistent_env.mli

```diff
@@ -106,13 +81,13 @@ type 'a sig_reader =
   -> flags:Cmi_format.pers_flags list
   -> 'a
 
-<<<<<<< HEAD
 (* If [add_binding] is false, reads the signature from the .cmi but does not
    bind the module name in the environment. *)
 (* CR-someday lmaurer: [add_binding] is apparently always false, including in the
    [-instantiate] branch. We should remove this parameter. *)
 val read : 'a t -> 'a sig_reader
-  -> Compilation_unit.Name.t -> filepath -> add_binding:bool -> Subst.Lazy.signature
+  -> Compilation_unit.Name.t -> Unit_info.Artifact.t -> add_binding:bool
```

Comment:
If I'm reading correctly from the conflict above, this should be

```suggestion
  -> Unit_info.Artifact.t -> add_binding:bool
```

(which is to say, the `Unit_info.Artifact.t` is replacing both the `modname` parameter and the `filepath` parameter). Seems reasonable, since the `modname` should in fact be determined by the `filepath` (which is to say, the artifact).

Reply:
The change was correct as-is I think, as per https://github.com/ocaml-flambda/flambda-backend/wiki/Strategy-for-5.2.0-Unit_info-and-compunit.  Later we can consider putting `Compilation_unit` inside `Unit_info`.